### DISCUSSION TITLE
Energy shift of all the datasets that iteratively added from the first-principle calculation

### DISCUSSION THREAD

**Discussion by DXiming** [deepmodeling/deepmd-kit#2974] [Q&A] [answered]
### Summary

Energy shift of all the datasets that iteratively added from the first-principle calculation, while for the initial data, the prediction is well.

### DeePMD-kit Version

v2.2.0b1

### TensorFlow Version

2.11.0

### Details

Hi, I have some issues with the training procedure using multiple datasets generated from `dp-gen`. The results show good prediction with the initial dataset, while for the iteratively added datasets, like `"iter1-0"`, `"iter0-1"`,  and `"iter0-0"`, there is a shift for the predicted energy. The training curve shows that the training procedure seems separated for each dataset with abnormal fluctuation of some training steps. I have tried to merge all the datasets into one, but the problem still exists. Can you help me solve this problem? I really appreciate any help you can provide.

![image](https://github.com/deepmodeling/deepmd-kit/assets/110084705/18ba1f40-31e7-46e6-a280-153e09cf4788)
![image](https://github.com/deepmodeling/deepmd-kit/assets/110084705/5bd89c3a-8b28-4eb8-8b48-fc9f901e916f)

![image](https://github.com/deepmodeling/deepmd-kit/assets/110084705/ea3be129-0484-492d-b2f0-47dd53a2f55b)

[input.json](https://github.com/deepmodeling/deepmd-kit/files/13246817/input.json)

**Comment by njzjz** [comment, marked as answer]
You might check if you use the same first-principle level for initial data and added data or any different things. From Figure 1, it's obvious that the two data sets are different.

**Reply by DXiming** [reply]
Thanks for your suggestion. This discrepancy is due to the omission of the VDW correction of my first-principle calculation.
